### DISCUSSION TITLE
Drum Kit DMX module: samples missing

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#244] [General]
Hi, Just for info...
I just tried the DK DMX module (SV modular Drum kit). It seems like the samples are missing inside.

**Comment by falkTX** [comment]
for which OS, and which host, and which plugin format?

**Reply by subderground** [reply]
Vst2 Windows 11 64 bit Reaper 😅

**Comment by falkTX** [comment]
Drum Kit module does not have samples, the outputs are CV gates.
You need to connect those signals to something that plays audio based on a trigger / gate.
